resources/mediasearch-vue/components/GenericResult.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<div class="wbmi-mediasearch-generic-result">
+	<div class="wbmi-generic-result">
```

Comment:
I'd like to do whatever we can to shorten class names. I don't think `mediasearch` is needed here especially since we've talked about reusing these components in something like a suggested images UI on wp.

Reply:
I support this 💯 percent